metadata-jobs/mce-consumer-job/src/main/java/com/linkedin/metadata/restli/RestliServletConfig.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.metadata.restli;
+
+import com.datahub.auth.authentication.filter.AuthenticationFilter;
+import com.linkedin.entity.client.RestliEntityClient;
+import com.linkedin.gms.factory.auth.SystemAuthenticationFactory;
+import com.linkedin.parseq.retry.backoff.ExponentialBackoff;
+import com.linkedin.restli.client.Client;
+import com.linkedin.restli.server.RestliHandlerServlet;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.boot.web.servlet.ServletRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Primary;
+
+import java.net.URI;
+
+@Configuration
+@Import({SystemAuthenticationFactory.class})
+public class RestliServletConfig {
```

Comment:
Are we spinning up an entire servlet in the mce consumer?

Reply:
Yeah, this is probably worth talking about at like a lunch and learn/project updates space since it's a non-trivial architectural change in general. We have talked about the goal of this inside the platform team at length.

Basically this is enabling the offloading of ingestion onto just MCE Consumer in cases where we are using Async ingestion. I.e. if we're using GMS to just bounce back to MCE Consumer by generating a MCP, why not just directly hit MCE Consumer to take network load off of GMS?

Reply:
Exactly, this is the core idea, remove GMS as the bottleneck for async ingestion and decouple the UI load from ingestion load.